docker-compose-common.yaml

```diff
@@ -59,6 +59,8 @@ services:
       - CABOT_HEADLESS
       - RMW_IMPLEMENTATION
       - CYCLONEDDS_URI
+      - CYCLONEDDS_NETWORK_INTERFACE_NAME
+      - CYCLONEDDS_NETWORK_INTERFACE_AUTODETERMINE
```

Comment:
Do we need this variable `CYCLONEDDS_NETWORK_INTERFACE_AUTODETERMINE` to be specified here? If yes, please describe it in the README. You specify the value in the launch script, so we may not need this here. It will be overridden anyway.

Reply:
`CYCLONEDDS_NETWORK_INTERFACE_AUTODETERMINE` is used when loading XML file specified by `CYCLONEDDS_URI` in docker, so we need to specify here.

When network interface name is set, `CYCLONEDDS_NETWORK_INTERFACE_AUTODETERMINE` should be `false`.
When network interface name is not set, `CYCLONEDDS_NETWORK_INTERFACE_AUTODETERMINE` should be `true`. 

Adding the variable which is set automatically may be confusing,  so I added following comment in docker-compose-common.xml
```
- CYCLONEDDS_NETWORK_INTERFACE_AUTODETERMINE # set automatically by launch.sh
```

Do we need to write README about variable which will be automatically set?